airflow/providers/amazon/aws/operators/sagemaker.py

```diff
@@ -419,6 +426,8 @@ def __init__(
         self.wait_for_completion = wait_for_completion
         self.check_interval = check_interval
         self.max_ingestion_time = max_ingestion_time
+        self.check_if_job_exists = check_if_job_exists
+        self.action_if_job_exists = action_if_job_exists
```

Comment:
Add a check here to make sure the value is something you support. See the same feature [on L658](https://github.com/apache/airflow/pull/25263/files#diff-78df59ac1d8db4b424ba03ab52a1215e1f85dfd41a4f436c3d60bd1bcd4c4273R658-R664)